tools/cockpit.spec

```diff
@@ -454,11 +454,16 @@ Requires: glib2 >= 2.50.0
 Conflicts: firewalld < 0.6.0-1
 Recommends: sscg >= 2.3
 Recommends: system-logos
-Requires: systemd >= 235
 Suggests: sssd-dbus
+%if !0%{?suse_version}
```

Comment:
Do you mind flipping this condition, so that we keep all the suse special cases in the same style?

Reply:
sure. will do